backend/app/Http/Controllers/AuthenticateController.php

```diff
@@ -13,6 +13,7 @@
 use Illuminate\Support\Facades\Cookie;
 use Illuminate\Support\Str;
 use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Hash;
```

Comment:
why do we have this duplicated in this controller?

Reply:
made changes as requested